doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -54,7 +120,7 @@ Figure 1-1 below shows this at a high level.
 <p align="center"><img src="../figures/ch01_k8s_arch.png" alt="Kubernetes Architecture for NFV" title="Kubernetes Architecture for NFV" width="100%"/></p>
 <p align="center"><b>Figure 1-1:</b> Kubernetes Architecture for NFV</p>
 
-### Other thoughts
+### More thoughts
 - Kubernetes / CaaS should be recognised in ETSI NFV / MANO
```

Comment:
ETSI NFV MANO recognise Kubernetes CaaS:

- VNFC definition contains containers from the beginning
- Support for pods in Vnfd is progressing in NFVIFA(19)000499 (r6 at the moment, but I'm pretty sure it will increase)
- Several NWI-s were accepted in the topic after IFA029

Also we should follow the same approach with standardization bodies as with open source projects. If we think that something is missing we should go there and fix the issue. I suggest to add a list of missing items to the Gaps chapter with their assigned resources to fix the issue. If we end up with a too long list we might consider to open a liaison connection towards the ISG.

Reply:
Thanks - I've removed this part of the document, will add references to ETSI docs in due course.